spot-client/src/spot-tv/calendars/google.js

```diff
@@ -129,12 +134,21 @@ export default {
  * @param {Array<Object>} events - The calendar events to filter.
  * @param {string} calendarEmail - The email of the calendar configured to
  * display.
+ * @param {Array<string>} knownDomains - Whitelist of domains which can be used
+ * as meeting links.
  * @returns {Array<Object>}
  */
-function filterJoinableEvents(events = [], calendarEmail) {
+function filterJoinableEvents(events = [], calendarEmail, knownDomains) {
```

Comment:
is this currently only to be used by google calendar ? would it be possible to have the detection in the calendar service instead and handle any calendar ?

But on the other hand probably the backend calendar will have a dedicated field, so not sure. Maybe as opt-in ? If it doesn't make sense for outlook or if you don't feel like doing it's fine too.

Reply:
Oh I see it's used in outlook too.